### PR TITLE
FFI: add two empty implement FFI for compatibility

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: proxy_components/proxy_ffi/src/encryption_impls.rs [hunk 1]
```diff
@@ -165,3 +165,18 @@ pub extern "C" fn ffi_handle_link_file(
         )
     }
 }
+
```
Comment (CalvinNeo) [inline proxy_components/proxy_ffi/src/encryption_impls.rs:168]: Compatibility for whom?

Comment (Lloyd-Pottiger) [reply proxy_components/proxy_ffi/src/encryption_impls.rs:168]: cse, https://github.com/tidbcloud/cloud-storage-engine/pull/1167

File: proxy_components/proxy_ffi/src/raftstore_proxy_helper_impls.rs [hunk 2]
```diff
@@ -65,6 +65,8 @@ impl RaftStoreProxyFFIHelper {
             fn_handle_new_file: Some(ffi_handle_new_file),
             fn_handle_delete_file: Some(ffi_handle_delete_file),
             fn_handle_link_file: Some(ffi_handle_link_file),
+            fn_get_keyspace_encryption: Some(CloudStorageEngineFFI::ffi_get_keyspace_encryption),
```
Comment (CalvinNeo) [inline proxy_components/proxy_ffi/src/raftstore_proxy_helper_impls.rs:68]: Could we also put these two ffis into one struct?

Comment (Lloyd-Pottiger) [reply proxy_components/proxy_ffi/src/raftstore_proxy_helper_impls.rs]: done

**Review by CalvinNeo — APPROVED** [review]
lgtm

**Pull request merged**